### PR TITLE
Translation fixed.

### PR DESCRIPTION
**Pull request by omarita**
Fixed for a better coherence with the original version. Made slightly more readable.

### CODE REVIEW

**Comment by jboner** [comment]
@dotta Could you check the fixes for the Italian translation? Thanks.

File: README.it.md [hunk 1]
```diff
@@ -1,25 +1,23 @@
...
- e un uptime pari al 100%. L'unità di utilizzo dei dati è il Petabyte. Molto semplicemente, le esigenze delle applicazioni di oggi non possono ssere soddisfatte dalle architetture software del passato.
+Ciò avviene perchè i requisiti delle applicazioni software sono mutati drasticamente negli ultimi anni. Fino a pochi anni fa le applicazione di grandi dimensioni erano ancora caratterizzate dall'impiego di decine di server,da tempi di risposta nell'ordine dei secondi, ore di manutenzione offline e gigabyte di dati. Le applicazioni di oggi sono installate su ogni genere di device, da quelli mobili ai cluster cloud-based che utilizzano migliaia di processori multi-core. Gli utenti di queste applicazioni si aspettano tempi di risposta nell'ordine dei millisecondi e un uptime pari al 100%. L'unità di utilizzo dei dati è il Petabyte. Molto semplicemente, le esigenze delle applicazioni di oggi non possono essere soddisfatte dalle architetture software del passato.
```
Comment (dotta) [inline README.it.md]: Maybe `ieri` is better than `passato` in the sentence above?

File: README.it.md [hunk 2]
```diff
@@ -1,25 +1,23 @@
...
 
-Crediamo che sia necessario un approccio coerente all'architettura dei sistemi software, e crediamo che i suoi elementi chiave siano già individuabili singolarmente: siamo a favore di sistemi che siano Responsivi, Elastici e Orientati ai Messaggi. Li chiamiamo Sistemi Reattivi.
+Crediamo che sia necessario un approccio coerente all'architettura dei sistemi software, e crediamo che i suoi elementi chiave siano già individuabili singolarmente: siamo a favore di sistemi che siano Responsivi, Resilienti, Elastici e Orientati ai Messaggi. Li chiamiamo Sistemi Reattivi.
```
Comment (dotta) [inline README.it.md]: I'd change `siamo a favore di` with `vogliamo`. Wdyt?

File: README.it.md [hunk 3]
```diff
@@ -1,25 +1,23 @@
...
 
-I Sistemi Reattivi sono flessibili, a basso accoppiamento e [scalabili](/it/glossary#Scalabilità): ciò li rende più semplici da sviluppare e malleabili al cambiamento. Tali sistemi sono di gran lunga più tolleranti ai [guasti](/it/glossary#Guasto) e reagiscono ad essi in modo elegante e non brusco. I Sistemi Reattivi sono altamente responsivi, offrendo agli [utenti](/it/glossary/#Utente) un feedback concreto e interattivo.
+I Sistemi Reattivi sono flessibili, a basso accoppiamento e [scalabili](/it/glossary#Scalabilità): ciò li rende più semplici da sviluppare e malleabili al cambiamento. Tali sistemi sono di gran lunga più tolleranti ai [guasti](/it/glossary#Guasto) e, quando questi si verificano, reagiscono con eleganza piuttosto che con un disastro. I Sistemi Reattivi sono altamente responsivi, offrendo agli [utenti](/it/glossary/#Utente) un feedback concreto e interattivo.
```
Comment (dotta) [inline README.it.md]: I'm not convinced `reagiscono con eleganza piuttosto che con un disastro` reads better than before. Maybe `guasto` is better than `disastro`, considering also it's coherent with what follows next. But maybe there is a better way of expressing it.

@cunei, please chime in

File: README.it.md [hunk 4]
```diff
@@ -1,25 +1,23 @@
...
+* <a name="Orientati ai Messaggi"></a>**Orientati ai Messaggi**: I Sistemi Reattivi si basano sullo [scambio di messaggi](/it/glossary#Orientamento-ai-messaggi) [asincrono](/it/glossary#Asincrono) per stabilire un confine tra i compoenti che possa garantirne il basso accoppiamento con gli altri, l'isolamento e la [trasparenza sul dislocamento](/it/glossary#Trasparenza-sul-dislocamento) e permetta di esprimere i [guasti](/it/glossary#Guasto) del componente sotto forma di messaggi. L'utilizzo di uno scambio esplicito di messaggi permette gestibilità del carico di lavoro, l'elasticità e il controllo del flusso calibrando e monitorando le code messaggi nel sistema ed applicando il [feedback di pressione](/it/glossary#Feedback-di-pressione) laddove necessario. Uno scambio di messaggi trasparente rispetto al dislocamento rende possibile, ai fini della gestione dei guasti, l'utilizzo degli stessi costrutti e semantiche sia su cluster che su singoli host. Uno stile di comunicazione [non bloccante](/it/glossary#Non-bloccante) fa sì che l'entità ricevente utilizzi le [risorse](/it/glossary#Risorsa) solo quando è attiva, il che porta ad un minor carico sul sistema.
```
Comment (dotta) [inline README.it.md]: `permette gestibilità del carico di lavoro` << syntax error

**Comment by jboner** [comment]
@omarita Thank a lot for the translation. Could you fix the issues so I can merge it?

**Comment by jboner** [comment]
@dotta Can you review the last changes? Thanks.

**Comment by jboner** [comment]
@dotta The original author of the PR is unresponsive. Is it good enough to merge as-is? Or should I close the PR?

**Comment by dotta** [comment]
@jboner It's good as is.

**Comment by jboner** [comment]
Thanks @dotta. Merging.

**Comment by jboner** [comment]
@dotta You mean that it can simply overwrite the existing one, as it is?

**Comment by dotta** [comment]
> @dotta You mean that it can simply overwrite the existing one, as it is?

@jboner No, I didn't mean that. I see there are conflicts and this PR can't be cleanly merged, so closing it may be best.

**Comment by jboner** [comment]
Good. Then let's do that. The author can always reopen it.

**Pull request closed without merging**